test-bin/echo/echo_bench.ml

```diff
@@ -0,0 +1,46 @@
+
+open Lwt.Infix
+
+open Capnp_rpc_lwt 
+
+let () =
+  Logs.set_level (Some Logs.Info);
+  Logs.set_reporter (Logs_fmt.reporter ())
+
+let rec create_for v = function
+  | 0. -> []
+  | n -> v n :: (create_for v (n-.1.))
+
+let run_client service = 
+  let n = 100000. in
+  let ops = n |> create_for (fun i -> 
+      fun () -> 
+        let%lwt res = Echo.ping service (Float.to_string i) in
+        Lwt.return (res = "echo:" ^ (Float.to_string i))
+    ) in
+  let st = Unix.gettimeofday () in
+  let%lwt _res = Lwt_list.map_p (fun v -> v ()) ops in
```

Comment:
I'd prefer to avoid using the old lwt ppx. `>>=` works fine, but if you really want a let syntax, you can use `let*` instead of `let%lwt` (requires OCaml 4.08 and the latest lwt for `Lwt.Syntax`).

Reply:
Awesome, I didn't know there was an official syntax yet for it. I had not wanted to use `let*` (manually defined) because if it wasn't the standard that would make things less clear. However given that there is an standard I can now use it! Will update to that.

Reply:
Gone with `>>=` for now to preserve backwards compatibility.